packages/functional-tests/src/validate-e2e.ts

```diff
@@ -0,0 +1,52 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import 'reflect-metadata';
+import { TestRunResult } from 'service-library';
+import { A11yServiceClient, A11yServiceCredential } from 'web-api-client';
+import * as yargs from 'yargs';
+
+// tslint:disable: radix no-any
+type Argv = {
+    clientId: string;
+    clientSecret: string;
+    authorityUrl: string;
+    waitTimeBeforeEvaluationInMinutes: string;
+    evaluationIntervalInMinutes: string;
+    releaseId: string;
+    baseUrl: string;
+};
+
+const argv: Argv = yargs.argv as any;
+
+console.log('[E2E] - start evaluate functional test results');
+
+const cred = new A11yServiceCredential(argv.clientId, argv.clientSecret, argv.clientId, argv.authorityUrl);
+const client = new A11yServiceClient(cred, argv.baseUrl);
+
+const waitTimeBeforeEvaluation = parseInt(argv.waitTimeBeforeEvaluationInMinutes) * 60 * 1000;
+const evaluationInterval = parseInt(argv.evaluationIntervalInMinutes) * 60 * 1000;
+
+let healthStatus: TestRunResult;
+
+setTimeout(async () => {
+    while (healthStatus !== 'pass') {
```

Comment:
we may need validation task timeout as well to terminate wait

Reply:
there is a timeout option in azure devops task config. We can use that.

Reply:
you should still have this fail after trying n times so that it fails gracefully instead of just throwing a timeout error.

Reply:
we are using a group task and timeout is hidden within group, better to have it explicitly in a code

Reply:
updated.